Wed/065_109_142_a_to_z_home_services.txt

```diff
@@ -35,8 +45,7 @@ All these changes should be reflected in this document only. So you can clone th
 By next week you should have one or two stories implemented.
 SRS document should be updated under srs_doc directory which is under parent of your project file in this repo
 
-16 Jan 2019 comments:
-Did not complete any story by this date
-SRS completed.
-Sent Pull request with priorities and dates updated. Also included SRS in pull request
```

Comment:
you have older version of the file and you are deleting 16 Jan 2019 comments unintentionally.

Please get latest from origin and merge them into your feature branch locally and then push changes to origin feature branch and then create pull request.